src/net/xhr/xhr.android.ts

```diff
@@ -0,0 +1,295 @@
+import { HTTPRequestMethods, IXHR, XMLHttpRequestEventTarget } from '.';
+
+import NativeEventEmitterComponent from '../../core/native-event-emitter-component';
+import { MobileOSProps } from '../../core/native-mobile-component';
+import { ResponseTypes, statuses, XMLHttpRequestResponseType } from './common';
+import { FormData } from './common';
+import { XHREventsEvents } from './xhr-events';
+
+const NativeXMLHttpRequest = requireClass('io.smartface.android.sfcore.net.XMLHttpRequest');
+
+export default class XHRAndroid<TEvent extends string = XHREventsEvents, TProps extends MobileOSProps = MobileOSProps>
+  extends NativeEventEmitterComponent<TEvent | XHREventsEvents, any, TProps>
+  implements IXHR
+{
+  public static UNSENT = 0;
+  public static OPENED = 1;
+  public static HEADERS_RECEIVED = 2;
+  public static LOADING = 3;
+  public static DONE = 4;
+
+  public onabort: (...args: any[]) => void = () => {};
```

Comment:
`public` key is redundant on Typescript, you can safely remove every public keyword